docs/tutorials/mastering.md

```diff
@@ -1,60 +1,624 @@
-| Machine | CPUs | RAM | Disk | Networks |
-| ------- | ---- | --- | ---- | -------- |
-| Control Plane Kubernetes Cluster     | 4 | 8g | 40g | `management` |
-| User Plane Kubernetes Cluster        | 4 | 4g | 20g | `management`, `access`, `core` |
-| Juju Controller + Kubernetes Cluster | 2 | 8g | 40g | `management` |
-| gNB Simulator Kubernetes  Cluster    | 4 | 4g | 20g | `management`, `ran` |
+In this tutorial, we will deploy and run the SD-Core 5G core network following Control and User Plane Separation (CUPS) principles. The radio and cell phone simulator will also be deployed on an isolated cluster. [Multipass](https://multipass.run/) is used to create separate VMs connected with [LXD](https://ubuntu.com/lxd) networking.
 
+## 1. Prepare Host
 
-## Sample Values
-
-For the purpose of this guide, the following values will be used.
+A machine running Ubuntu 22.04 with the following resources:
+  - At least one NIC with internet access
+  - 8 cores
+  - 24 GB RAM
+  - 150 GiB disk  
 
 ### Networks
 
+The following networks will be used to create test environment.
+
 | Name | Subnet | Gateway IP |
 | ---- | ------ | ---------- |
 | `management` | 10.201.0.0/24 | 10.201.0.1 |
 | `access`     | 10.202.0.0/24 | 10.202.0.1 |
 | `core`       | 10.203.0.0/24 | 10.203.0.1 |
 | `ran`        | 10.204.0.0/24 | 10.204.0.1 |
 
+
+Create local network bridges to be used by LXD by adding below configuration under `/etc/netplan/50-sdcore-networks.yaml`.
+
+```console
+cat << EOF | sudo tee /etc/netplan/50-sdcore-networks.yaml
+# /etc/netplan/50-sdcore-networks.yaml
+network:
+  bridges:
+    mgmt-br:
+      addresses:
+        - 10.201.0.14/24
+      routes:
+        - to: default
+          via: 10.201.0.1
+          metric: 110
+    access-br:
+      addresses:
+        - 10.202.0.14/24
+      routes:
+        - to: 10.204.0.0/24
+          via: 10.202.0.1
+    core-br:
+      addresses:
+        - 10.203.0.14/24
+      routes:
+        - to: default
+          via: 10.203.0.1
+          metric: 203
+    ran-br:
+      addresses:
+        - 10.204.0.14/24
+      routes:
+        - to: 10.202.0.0/24
+          via: 10.204.0.1
+
+  version: 2
+EOF
+```
+
+```console
+sudo netplan apply
+```
+
+### Install and Configure LXD
+
+Install LXD:
+```console
+sudo snap install lxd
+```
+
+```console
+cat << EOF | sudo tee ~/preseed.yaml
+config: {}
+networks: []
+storage_pools:
+- config:
+    size: 150GiB
+    source: /var/snap/lxd/common/lxd/disks/default.img
+    zfs.pool_name: default
+  description: ""
+  name: default
+  driver: zfs
+profiles:
+- config: {}
+  description: Default LXD profile
+  devices:
+    root:
+      path: /
+      pool: default
+      type: disk
+  name: default
+projects:
+- config:
+    features.images: "true"
+    features.networks: "true"
+    features.networks.zones: "true"
+    features.profiles: "true"
+    features.storage.buckets: "true"
+    features.storage.volumes: "true"
+  description: Default LXD project
+  name: default
```

Comment:
Wouldn't a dedicated project be better choice here? What if someone is already using LXD for something else?

Reply:
OK, Changing project name as the one below:
  description: SD-Core project
  name: sdcore